### PR TITLE
fixed BOX_PERMANENT_ID_USER3 collision

### PR DESCRIPTION
**Pull request by RomanLut**
BOXLOITERDIRCHN has the same id as BOX_PERMANENT_ID_USER3

Changed BOXLOITERDIRCHN in order to let BOX_PERMANENT_ID to have consecutive ids.

### CODE REVIEW

**Comment by JulioCesarMatias** [comment]
was caused by this PR >> https://github.com/iNavFlight/inav/commit/da521b0fdeb47d6d8de893fdb58622733a381e57

**Review by JulioCesarMatias — APPROVED** [review]
Thanks for the correction!

File: src/main/fc/fc_msp_box.c [hunk 1]
```diff
@@ -87,14 +87,14 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT + 1] = {
     { .boxId = BOXUSER1,            .boxName = "USER1",             .permanentId = BOX_PERMANENT_ID_USER1 },
     { .boxId = BOXUSER2,            .boxName = "USER2",             .permanentId = BOX_PERMANENT_ID_USER2 },
     { .boxId = BOXUSER3,            .boxName = "USER3",             .permanentId = BOX_PERMANENT_ID_USER3 },
-    { .boxId = BOXLOITERDIRCHN,     .boxName = "LOITER CHANGE",     .permanentId = 49 },
+    { .boxId = BOXLOITERDIRCHN,     .boxName = "LOITER CHANGE",     .permanentId = 57 }, //BOX_PERMANENT_ID_USER3 = 49
```
Comment (JulioCesarMatias) [inline src/main/fc/fc_msp_box.c]: I don't think we need the comment added...

Comment (RomanLut) [reply src/main/fc/fc_msp_box.c]: Personnaly I think that such comments help people to to avoid mistakes. But Ok, I removed it.

**Review by JulioCesarMatias — CHANGES_REQUESTED** [review]
instead of just changing the value from 49 to 57, you can reorder again... Change the value of "loiter change" to 50, "MSP RC OVERRIDE" to 51, "PREARM" to 52, and continue to the last ("Soaring")

**Comment by RomanLut** [comment]
Done as requested.

**Review by JulioCesarMatias — APPROVED** [review]
ok, thanks!

**Comment by MrD-RC** [comment]
This will need release notes, as all ranges for:
- `BOXLOITERDIRCHNANGE`
- `BOXMSPRCOVERRIDE`
- `BOXPREARM`
- `BOXTURTLE`
- `BOXNAVCRUISE`
- `BOXAUTOLEVEL`
- `BOXPLANWPMISSION`
- `BOXSOARING`

will be reassigned to the wrong switches and ranges. Any pilot using these modes will need to reconfigure them.

I think it would have been better to just mode `BOXLOITERDIRCHANGE` to the end of the array as 57. Loiter dir change is hardly used; whereas modes like Cruise and Auto level are used a lot. Not changing those IDs would have been better. To be honest, it may be worth submitting a new PR to do that.

**Comment by MrD-RC** [comment]
No problem. The new PR will still break the diff. But it will be a problem for less people 🙂

**Comment by JulioCesarMatias** [comment]
Do you remember my proposal to change the box number in the diff, by the name of the flight mode? This model of mine would certainly avoid this kind of thing... But unfortunately it makes F722 cards crash (despite working 100% on the F405)... I still have the algorithm in my fork.

**Pull request merged**